syn/yosys-uhdm/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = '%s_%04i_%s'|format(GIT_DESCRIBE_TAG|replace('v','') or '0.X.X', GIT_DESCRIBE_NUMBER|int,   GIT_DESCRIBE_HASH or 'gUNKNOWN') %}
+
+package:
+  name: yosys-uhdm
+  version: {{ version }}
+
+source:
+  git_rev: master
+  git_url: https://github.com/alainmarcel/uhdm-integration.git
+
+build:
+  number: {{ environ.get('DATE_NUM') }}
+  string: {{ environ.get('DATE_STR') }}
+  script_env:
+    - CI
+    - USE_SYSTEM_GCC_VERSION
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - python {{ python }}
+    - cmake
+    - pkg-config
+    - libuuid
+    - gperftools
+    - flex
+    - libunwind
+    - swig
+    - ncurses
+test:
+    requires:
+        - ncurses
+        - readline
+        - libffi
```

Comment:
With 99% certainty I'd say those 3 packages should be in `requirements: run` section if they are required to successfully call `yosys-uhdm --version`. Please add such section and remove this whole `requires`.

Reply:
Requirements moved